lib/github/client.js

```diff
@@ -32,87 +32,101 @@
 
 'use strict';
 
-const util = require('util');
-
 const Gofer = require('gofer');
 
+const pkg = require('../../package');
+
 const parseRepository = require('./parse-repository');
 
-function Github(config) {
-  Gofer.call(this, config, 'github');
-}
+class Github extends Gofer {
+  constructor(config) {
+    super(config, 'github', pkg.version);
```

Comment:
```suggestion
    super(config, 'github', pkg.version, pkg.name);
```

(as long as we've got `pkg` handy anyway...)